srcts/src/shiny/index.ts

```diff
@@ -606,6 +606,24 @@ class ShinyClass {
       sendOutputHiddenState
     );
 
+    function doSendWindowSize() {
+      inputs.setInput(".clientdata_window_width", window.innerWidth);
+      inputs.setInput(".clientdata_window_height", window.innerHeight);
+      inputs.setInput(
+        ".clientdata_scroll_width",
+        document.documentElement.scrollWidth
+      );
+      inputs.setInput(
+        ".clientdata_scroll_height",
+        document.documentElement.scrollHeight
+      );
+    }
+
+    sendImageSizeFns.setImageSend(inputBatchSender, doSendWindowSize);
```

Comment:
Unfortunately, I don't think `.setImageSend()` was designed to be used this way (i.e., it's only designed to have one callback, not multiple). 

I have a couple ideas on how we could proceed -- probably best if we talk through it in person